packages/components/image-viewer/src/image-viewer.vue

```diff
@@ -308,12 +309,12 @@ function reset() {
 function toggleMode() {
   if (loading.value) return
 
-  const modeNames = Object.keys(Mode)
-  const modeValues = Object.values(Mode)
+  const modeNames = keysOf(modes)
+  const modeValues = Object.values(modes)
```

Comment:
**Question:** why not:

```ts
const [modeNames, modeValues] = Object.entries(modes)
```

Reply:
The results of `entries` is `[[name, value], [name, value]]`😅

Reply:
Oh right, my bad.